configs/automations.yaml

```diff
@@ -379,17 +379,7 @@ automations:
             description: Test the feature
             due_date_days_difference: 2
   - _target_: todoist.automations.activity.Activity
-    name: Activity Last Week
-    nweeks: 1
-  - _target_: todoist.automations.activity.Activity
-    name: Activity 2 Weeks
-    nweeks: 2
-  - _target_: todoist.automations.activity.Activity
-    name: Activity Last Month
-    nweeks: 4
-  - _target_: todoist.automations.activity.Activity
-    name: Activity Last 10 years
-    nweeks: 520
+    name: Activity Adaptive Fetch
```

Comment:
Maybe also increase default parameters  (windows size + early stopping) 
in config